opta/commands/init_templates/environment/aws/opta.yaml

```diff
@@ -0,0 +1,26 @@
+name: ""
+org_name: ""
+providers:
+  aws:
+    region: ""
+    account_id: ""
+modules:
+  - type: base
+  - type: dns
+    delegated: false
+  - name: deployeruser
```

Comment:
don't need this-- this creates an admin user. We can start prompting folks to create users later

Reply:
Ah, thank you for pointing this out!

Reply:
then pls remove

Reply:
whoops -- didn't realize i didn't push

Reply:
c61a0a2f4b017d0cadd2fbe8070cc0f0e94d33ec